tests/tests/swfs/avm2/textfield_getlinetext/output.txt

```diff
@@ -0,0 +1,8 @@
+///
+hello world
+RangeError: Error #2006: The supplied index is out of bounds.
```

Comment:
This is not the Flash-matching error message, Flash provides some additional information about the index- can you just print the error class name and code?

Reply:
Interesting, I can do that. What error do you get? This is from Flash 32,0,0,314.

Reply:
Maybe I'm mistaken- what I remember is that we omitted the index info from one of the index errors (or maybe it was a different error?)

Reply:
I think you might be thinking of the exact opposite, we produce a more exact error message, while Flash has just the generic out of bounds message: https://github.com/ruffle-rs/ruffle/blob/0df6bafdaf9cbbfcf0b1bb395f77d684c1d96cef/core/src/avm2/globals/flash/display/display_object_container.rs#L334.